components/concourse-terraform-resource/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # pinned to a beta-backend version of ljfranklin/terraform-resource
 # FIXME: once beta-backend is promoted to master, use propper version labels
-FROM ljfranklin/terraform-resource@sha256:688b879952a041c176087973bdefb6d40bc14a0028f69d4576764d8b2b67501d
+FROM ljfranklin/terraform-resource@sha256:15eee04112da38c0fcbdb9edb86a6b5acff4a800f21cb29b4e30dc58b27e5d0d
```

Comment:
does this invalidate the `FIXME` above? The comment is from 6301d5f5007dadf558f076cf7482f5dc1617eead cc @chrisfarms

Reply:
Not sure. There is a beta-backend tag but it doesn't have a hash matching the old one above. @chrisfarms? https://github.com/ljfranklin/terraform-resource/pull/56#issuecomment-402568589 looks related. Is it https://github.com/ljfranklin/terraform-resource/commits/WIP-tf-backends ? Maybe it has been merged - https://github.com/ljfranklin/terraform-resource/commit/ca2e6ac5886790fb2b303811a0c3457ded287f99 ?

Reply:
yeah the beta-backend stuff is on master now ... so we should be safe to use the digest for `0.12.12` (`15eee04112da38c0fcbdb9edb86a6b5acff4a800f21cb29b4e30dc58b27e5d0d`) image and drop the comment